src/utils/authUtils.ts

```diff
@@ -16,6 +17,29 @@ export const setAuthErrorMessage = (
   errorMessage: string,
 ) => dispatch({ type: 'SET_ERROR_MESSAGE', errorMessage });
 
+export const setAuthSuccessMessage = (
+  dispatch: AuthDispatch,
+  successMessage: string,
+) => dispatch({ type: 'SET_SUCCESS_MESSAGE', successMessage });
+
+export const forgotPassword = async (
+  dispatch: AuthDispatch,
+  params: { email: string },
+) => {
+  const auth = getAuth();
+  await sendPasswordResetEmail(auth, params.email)
+    .then(() => {
+      setAuthSuccessMessage(dispatch, 'Email sent. Check your inbox to reset.');
+      setAuthErrorMessage(dispatch, '');
+    })
+    .catch(error => {
+      // eslint-disable-next-line no-console
+      console.warn('(forgotPassword) error', error);
```

Comment:
Remove this console.warn since we're already catching the error using setAuthErrorMessage.